internal/controller/apiextensions/claim/configurator.go

```diff
@@ -60,6 +61,16 @@ func NewAPIDryRunCompositeConfigurator(c client.Client) *APIDryRunCompositeConfi
 	return &APIDryRunCompositeConfigurator{client: c}
 }
 
+func withoutReservedK8sEntries(a map[string]string) map[string]string {
```

Comment:
nit: Should better be moved below `func (c *APIDryRunCompositeConfigurator) Configure` instead of appearing on top.

Reply:
done, take a look.